src/Files/MultilingualResources/Files.vi.xlf

```diff
@@ -2928,83 +2928,83 @@
         </trans-unit>
         <trans-unit id="DateFormat" translate="yes" xml:space="preserve">
           <source>Date format</source>
-          <target state="new">Date format</target>
+          <target state="new">Định dạng ngày tháng</target>
```

Comment:
Can you update the state to `translated`? https://files.community/docs/contributing/translating-files

Reply:
Thanks, I changed them to `translated`